packages/myst-to-typst/src/index.ts

```diff
@@ -454,6 +456,23 @@ const handlers: Record<string, Handler> = {
     state.write(']');
     state.ensureNewLine();
   },
+  card(node, state) {
+    if (node.url) {
+      node.children?.push({ type: 'paragraph', children: [{ type: 'text', value: node.url }] });
+    }
+    state.renderChildren(node);
+    state.ensureNewLine();
+    state.write('\n');
+  },
+  cardTitle(node, state) {
+    headingHandler({ ...node, depth: 3 }, state);
```

Comment:
I think this should just be bold here. Otherwise all of these will end up in  table of contents, etc.

Reply:
Ah yeah, that `depth 3` was also totally arbitrary. Thanks